test_runner/regress/test_tenant_conf.py

```diff
@@ -336,10 +336,15 @@ def test_live_reconfig_get_evictions_low_residence_duration_metric_threshold(
 ):
     neon_env_builder.enable_pageserver_remote_storage(RemoteStorageKind.LOCAL_FS)
 
-    env = neon_env_builder.init_start()
+    env = neon_env_builder.init_start(
+        initial_tenant_conf={
+            # disable compaction so that it will not download the layer for repartitioning
```

Comment:
You wanted to expand this comment but I didn't want to include it. Will edit soon.

Reply:
https://github.com/neondatabase/neon/pull/5108/files/8692af2726a29a8d3455ecb2b391e6a77d48349e#r1403234714